ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Impl/Common.hs

```diff
@@ -537,7 +538,7 @@ data TraceEventMempool blk
       MempoolSize
       -- ^ The current size of the Mempool.
   | TraceMempoolManuallyRemovedTxs
-      [GenTxId blk]
+      (NE.NonEmpty (GenTxId blk))
```

Comment:
Is having at least one `GenTxid` here the motivation for this change?

Reply:
Exactly

Reply:
For ref https://github.com/IntersectMBO/ouroboros-consensus/pull/305